src/Paket.Core/ProjectFile.fs

```diff
@@ -941,6 +941,9 @@ module ProjectFile =
                 project.ProjectNode.AppendChild analyzersNode |> ignore
             )
 
+    let touch (project:ProjectFile) =
+        if File.Exists(project.FileName) then
```

Comment:
I was a little bit incorrect in my wording, `touch` was intended to **replace** the `save true` call.

But you're right, why should it be there in first place?
The project file should know the own state, whether it's changed or not, right? So `save true` could actually utilize this knowledge to decide whether to simply change the last access date or write out the contents.